salt/kubernetes-master/controller-manager.jinja

```diff
@@ -10,7 +10,8 @@ KUBE_CONTROLLER_MANAGER_ARGS="\
     --cluster-cidr={{ pillar['cluster_cidr'] }} \
     --service-account-private-key-file={{ pillar['ssl']['key_file'] }} \
     --root-ca-file={{ pillar['ssl']['ca_file'] }} \
-{% if salt['pillar.get']('infrastructure', 'libvirt') == 'aws' -%}
```

Comment:
I guess we should leave this in, and switch to the cloud:provider pillar? (From memory, AWS does not require any specific configuration)